tools/ci_build/github/linux/docker/Dockerfile.ubuntu_gpu_training

```diff
@@ -26,3 +31,29 @@ ARG BUILD_UID=1000
 RUN adduser --gecos 'onnxruntime Build User' --disabled-password $BUILD_USER --uid $BUILD_UID
 WORKDIR /home/$BUILD_USER
 USER $BUILD_USER
+
+ARG MINICONDA_PREFIX=/home/$BUILD_USER/miniconda3
+RUN if [ "$USE_CONDA" = true ] ; \
+    then MINICONDA=miniconda.sh && \
+    wget --no-check-certificate --no-verbose https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O $MINICONDA && \
+    chmod a+x $MINICONDA && \
+    ./$MINICONDA -b -p $MINICONDA_PREFIX && \
+    rm ./$MINICONDA && \
+    $MINICONDA_PREFIX/bin/conda clean --yes --all && \
+    $MINICONDA_PREFIX/bin/conda install -y python=$PYTHON_VERSION && \
+    $MINICONDA_PREFIX/bin/conda install -c conda-forge -y protobuf ; \
```

Comment:
If pipeline fail, test without this protobuf install

Reply:
Could you please remove the "--no-check-certificate" ? It looks unsafe

Reply:
Is it possible to fix the version of Miniconda3?

Reply:
ok, I'll do that.